src/ISM330DLCSensor.cpp

```diff
@@ -707,7 +707,7 @@ ISM330DLCStatusTypeDef ISM330DLCSensor::Set_G_ODR_When_Enabled(float odr)
 {
   ISM330DLC_ACC_GYRO_ODR_G_t new_odr;
 
-  new_odr = ( odr <=  13.0f )  ? ISM330DLC_ACC_GYRO_ODR_G_13Hz
+  new_odr = ( odr <=  13.0f )  ? ISM330DLC_ACC_GYRO_ODR_G_12Hz5
```

Comment:
we need also to modify the condition in this way: "new_odr = ( odr <= 12.5f ) ? ISM330DLC_ACC_GYRO_ODR_G_12Hz5"